packages/blade/src/components/Chip/AnimatedChip.native.tsx

```diff
@@ -0,0 +1,53 @@
+import getIn from 'lodash/get';
+import React from 'react';
+import styled from 'styled-components/native';
+import Animated, { useAnimatedStyle, useSharedValue, withTiming } from 'react-native-reanimated';
+import { getAnimatedChipStyles } from './getAnimatedChipStyles';
+import type { AnimatedChipProps } from './types';
+import { chipMotionTokens } from './chipTokens';
+import { useTheme } from '~components/BladeProvider';
+
+const StyledAnimatedChip = styled(Animated.View)<AnimatedChipProps>((props) => {
+  return {
+    ...getAnimatedChipStyles(props),
+    alignSelf: 'center',
+  };
+});
+
+const AnimatedChip = ({
+  borderColor,
+  children,
+  isPressed,
+  isDisabled,
+}: Omit<AnimatedChipProps, 'theme'>): React.ReactElement => {
+  const { theme } = useTheme();
+  const scaleDownAnimation = useSharedValue(1);
+
+  const easing = getIn(theme, chipMotionTokens.timingFunction);
+  const duration = getIn(theme, chipMotionTokens.duration);
+
+  React.useEffect(() => {
+    scaleDownAnimation.value = withTiming(isPressed ? 0.92 : 1, {
+      duration,
+      easing,
+    });
+  }, [isPressed, easing, duration, scaleDownAnimation]);
+
+  const chipAnimation = useAnimatedStyle(() => {
+    return {
+      transform: [
+        {
+          scale: scaleDownAnimation.value,
+        },
+      ],
+    };
+  }, []);
```

Comment:
```jsx
  const chipAnimation = useAnimatedStyle(() => {
    return {
      transform: [
        {
          scale: withTiming(isPressed ? 0.92 : 1, {
            duration,
            easing,
          }),
        },
      ],
    };
  }, [isPressed]);
```

Just this would work, you can remove the useEffect stuff